survey-runner-queue/message_queue.tf

```diff
@@ -20,6 +20,7 @@ resource "aws_instance" "rabbitmq" {
 
   tags {
     Name = "${var.env}-rabbitmq-${count.index + 1}"
+    Backup = "RabbitMQ"
```

Comment:
Instead of the name `Backup` we could make this `Service`? it makes it a lot more reusable?